news/2018-12-02-project-loved-week-of-december-2nd.md

```diff
@@ -0,0 +1,253 @@
+
+[![](/wiki/shared/news/2018-12-02-project-loved-week-of-december-2nd/mania/hellfire.jpg)](https://osu.ppy.sh/community/forums/topics/837122)
+
+[Hommarju - Hellfire](https://osu.ppy.sh/beatmapsets/469343#mania) by [snoverpk](https://osu.ppy.sh/users/3595196) and [\_underjoy](https://osu.ppy.sh/users/2235750)  
+*written by [TheToaphster](https://osu.ppy.sh/users/7616811)*
+
+It's all about how well one can handle extremely fast jacks in the next nomination for osu!mania. Clocking in at 176 BPM with a large surplus of fast jack-tech is snoverpk's Hellfire. This chart starts with about 20 seconds of calm before unleashing a fury of minijacks and gluts with the occasional burst to spice things up a bit. With a small break and some buildup, it comes to a massive wall, which is only to be followed by the densest section of the chart to finish it off. This is an intense challenge to be sure, only shown off by its inclusion in the osu!mania 4K World Cup 2016 Finals map pool. This and the great use it has as a skill tester make it a fantastic nomination for the Loved Section.
+
+[![](/wiki/shared/news/2018-12-02-project-loved-week-of-december-2nd/mania/satori-de-pon.jpg)](https://osu.ppy.sh/community/forums/topics/837123)
+
+[goreshit - satori de pon!](https://osu.ppy.sh/beatmapsets/400292#mania) by [qodtjr](https://osu.ppy.sh/users/3021168)  
+*written by [stupud man](https://osu.ppy.sh/users/2141612)*
+
+It's been awhile since qodtjr has made an appearance in Project Loved, and he's back with arguably his most popular map yet. satori de pon!'s "easier" difficulty is more than enough to give higher-level players a challenge. Its unique anchor-heavy chordstreaming at such a high BPM makes it one of the only speed chordstream maps at its difficulty level. Although chordstreams are more than common for qodtjr to map, it's pleasantly surprising to see him take it on in this manner. Its recently updated harder difficulty takes that same chart and layers it even further, creating that challenge scaled to the playing level of the people too inhuman to play this game. While it might seem too hard to approach at first with its underrated star rating, it's definitely something that has made its way into multiple players' practice routines, and is more than worthy of a leaderboard.
+
+[![](/wiki/shared/news/2018-12-02-project-loved-week-of-december-2nd/mania/sore-wa-chiisana-hikari-no-youna-feat-k-chan-dj-jo-remix.jpg)](https://osu.ppy.sh/community/forums/topics/837124)
+
+[dj-Jo - Sore wa Chiisana Hikari no Youna feat. k\*chan [ dj-Jo Remix ]](https://osu.ppy.sh/beatmapsets/477820#mania) by [ArcherLove](https://osu.ppy.sh/users/2468021)  
```

Comment:
the name here is different than the one in the beatmap page... was the one in the beatmap page filled in wrong?